main/cogs/ladder.py

```diff
@@ -40,7 +63,184 @@ async def ladder(self, ctx, mode="off"):
+
+                message += str(pos) + "." + buffer1 + user["username"] + buffer2 + str(round(user["adjusted_rating"])) \
+                           + buffer3 + str(user["num_wins"]) + "-" + str(user["num_losses"]) + "\n"
+                if pos % 50 == 0:
+                    message += "```"
+                    await ctx.send(message)
+                    message = "```"
+                pos += 1
+            message += "```"
+
+            if message != "``````":
+                await ctx.send(message)
+        else:
+            embed = discord.Embed(color=0xEA7D07)
+            embed.add_field(name='The !ladder command must be used here:', value=f'<#{ev.get_var("bot_spam_channel_id")}>')
+            await ctx.send(embed=embed)
+
+    @commands.command(name="ladderTiers", help="Display the ladder with tiers and a minimum of 5 games")
+    @commands.cooldown(1, 300, commands.BucketType.default)
+    async def ladder_tiers(self, ctx, mode="off"):
+        if str(ctx.channel.id) == ev.get_var("bot_spam_channel_id"):
+            mode = ladders.find_game_mode(mode)
+
+            ladder_values = sorted(ladders.ladders[mode].values(), key=lambda x: x["adjusted_rating"], reverse=True)
+
+            users_by_tier = []
+
+            current_tier = 0
+            user_count = 0
+            tier_messages = []
+            for index, user in enumerate(ladder_values):
+                if (user["num_wins"] + user["num_losses"]) < 5:
+                    continue
+
+                user_count += 1
+                buffer1 = " " * (4 - len(str(user_count)))
+                buffer2 = " " * (18 - len(user["username"]))
+                buffer3 = " " * (7 - len(str(round(user["adjusted_rating"]))))
+                message = str(user_count) + "." + buffer1 + user["username"] + buffer2 + str(round(user["adjusted_rating"])) \
+                           + buffer3 + str(user["num_wins"]) + "-" + str(user["num_losses"]) + "\n"
+
+                while user["adjusted_rating"] < tier_min_elo[current_tier]:
+                    users_by_tier.append(tier_messages[:])
+                    tier_messages.clear()
+                    current_tier += 1
+
+                tier_messages.append(message)
+
+            emojis = ctx.message.guild.emojis
+            line_count = 1
+            tier_message = "**" + mode + " Ladder Tiers**\n"
+            index = 0
+            for tier in users_by_tier:
+                if len(tier) > 0:
+                    tier_captain = next((e for e in emojis if e.name.lower() == str(tier_captains[index].value).replace(" ", "").lower()), "")
+                    tier_message += f"** {tier_captain} {str(tier_min_elo[index])}+**"
+                    tier_message += " \n```"
+                    for line in tier:
+                        tier_message += line
+                        line_count += 1
+                    tier_message += "```\n"
+                    line_count += 2
+                if line_count > 40:
```

Comment:
Doesn't really look like it but it would be nice if there was a way to have slightly less gap between the bottom of each "code block" and the line that begins the next tier